src/app/auth/error/page.tsx

```diff
@@ -0,0 +1,32 @@
+import { Icons } from "@/components/Icons";
+import Link from "next/link";
+import { buttonVariants } from "@/components/ui/button";
+
+const Page = () => {
+  return (
+    <>
```

Comment:
Why are you using `Fragment` here? And other places

Reply:
Not needed, fixed in new commit, Thank you